src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Issue**: The condition in the loop incorrectly compares the elements directly rather than their absolute values. This change deviates from the method's purpose of finding the absolute maximum. <br> **Fix**: Restore the comparison of absolute values within the loop to ensure the function correctly calculates the absolute maximum. <br> **Code Suggestion**: 
 ```
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```